ouroboros-network/src/Ouroboros/Network/NodeToNode.hs

```diff
@@ -13,8 +14,12 @@ module Ouroboros.Network.NodeToNode (
   , NodeToNodeVersionData (..)
   , DictVersion (..)
   , nodeToNodeCodecCBORTerm
+
+  , nodeToNodeConnectTo
+  , withServerNodeToNode
```

Comment:
Since this module is supposed to become the top level high level API for users of the node -> node protocol, then I think we should use short names, and if necessary use qualified names internally. There would be no need for clients of this module to use qualified names since they would not also be importing the internal lower level modules.

So I'd just use
```
connectTo
withServer
```

Reply:
Good suggestion, I'll fix that.

Reply:
I'll rename the current `connectTo` as `connectToNode` so both general one will have consistent names (`connectToNode` and `withServerNode`) and nothing will clash.